os/arch/arm/src/armv7-m/up_memfault.c

```diff
@@ -148,6 +148,9 @@ static inline void print_memfault_detail(uint32_t *regs, uint32_t cfsr, uint32_t
 
 int up_memfault(int irq, FAR void *context, FAR void *arg)
 {
+	/* Add new line to distinguish between normal log and assert log.*/
+	lldbg("\n");
```

Comment:
fault has the irqsave at start of the function. Let's do this after the irqsave.